neard/src/runtime/mod.rs

```diff
@@ -193,6 +197,11 @@ impl NightshadeRuntime {
         epoch_manager.get_epoch_info(&epoch_id).map(|info| info.epoch_height()).map_err(Error::from)
     }
 
+    fn get_epoch_id(&self, parent_hash: &CryptoHash) -> Result<EpochId, Error> {
```

Comment:
`get_epoch_from_prev_block` is already a method in `RuntimeAdapter`

Reply:
Yes, but I need epoch of prev block, and I do not have hash of block before prev

Reply:
I see. Then `parent_hash` is not the best name :)

Reply:
Fixed